src/legacy/core_plugins/kibana/public/management/sections/settings/components/field/field.js

```diff
@@ -565,6 +567,26 @@ export class Field extends PureComponent {
 
   renderDescription(setting) {
     let description;
+    let deprecation;
+
+    if (setting.deprecation) {
+      deprecation = (
+        <>
+          <EuiToolTip content={setting.deprecation.message}>
+            <EuiBadge
+              color="warning"
+              onClick={() => {
+                window.open(setting.deprecation.url, '_blank');
```

Comment:
nit: Not sure if `EuiBadge` allows it, but is this possible to have this link as `a` tag? (to get right click context menu)

Reply:
@cchaos ?

Reply:
Not currently sorry. It only supports `onClick` behavior and not `href`. We enforce only buttons to use `onClick` and links to use `href`.